samples/update_column.bal

```diff
@@ -0,0 +1,27 @@
+import ballerina/log;
```

Comment:
Need to add licence header

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerinax-microsoft.excel/pull/10/commits/1e5488e40502173708a9034d8ef847032e53384a